### ISSUE TITLE
I cannot edit my profile or remove my posts 

### ISSUE THREAD

**Issue by DevilEars** [forem/forem#8759]
**Describe the bug**
Cannot update profile, or delete posts. I do not have a twitter account, I logged in via Github. I have logged out and logged in again, the same issue persists.

**To Reproduce**
This is for posts:
1. Go to 'https://dev.to/settings' 
1. Delete content in the **Summary** section. Note that this is just one way to reproduce the bug. I can delete anything on my profile, and the same bug results.
1. Click on the _Save_ button
1. See this error being displayed:
![image](https://user-images.githubusercontent.com/49192883/84999697-cabbe400-b151-11ea-8a80-7abfa35c9793.png)

**Expected behavior**
The profile should be updated without error messages 

**Desktop (please complete the following information):**
- OS: Windows 10
- Browser: Opera
- Version: 68.0.3618.125

**Additional context**
When I try to update my blog posts, I get an HTTP 500 error. I suspect that this is related, but it could be a separate issue:
![image](https://user-images.githubusercontent.com/49192883/85000133-4e75d080-b152-11ea-962b-a5ad2c2a5fe9.png)
![image](https://user-images.githubusercontent.com/49192883/85002758-98ac8100-b155-11ea-9ff7-b1ac56c89594.png)

**Comment by citizen428** [comment]
Thanks for opening an issue! Is this for https://dev.to/DevilEars? Can you provide a bit more info (how do you sign in, what happens when you try to update a post, etc)? In the meantime feel free to email you@dev.to if you want posts removed.

**Comment by DevilEars** [comment]
Thanks for the prompt response!
Yes, some woketard flagged my profile now I cannot edit my profile, or remove my posts, or do pretty much anything. This may be relevant information. Whenever I try to update anything on my profile, I get an HTTP 500 service error. 

l'll try to be more clear and give you a step by step way to recreate this scenario. I have already emailed you@dev.to and they haven't been of much help.

**Comment by DevilEars** [comment]
Kindly note that I updated the original post with more information

**Comment by DevilEars** [comment]
By the way, this issue is also present in Brave browser, running in a private tab to rule out stale cookies or other browser data hanging around.

**Comment by citizen428** [comment]
Thanks for providing extra information! Have you been able to solve this through your interactions with yo@dev.to by now?

**Comment by DevilEars** [comment]
No, I haven't received any more information from them. I did email them again, notifying them that I did log a bug for what I view as a bugs: I should be able to edit my own profile, and my own content, regardless of who put me in the sin bin - if I even am in the sin bin. I am speculating that the bugs we see here are side-effects because the flagging procedure is not exactly well thought through by Dev.to, which seems to be a pattern on this platform. But no, yo@dev.to has not contacted me since 17 June.
